rbc/src/test/java/com/library/rbc/integration/bookcontroller/IntegrationTest.java

```diff
@@ -13,58 +16,61 @@
 @SpringBootTest(webEnvironment = RANDOM_PORT)
 public class IntegrationTest {
 
-  @Autowired
-  private WebTestClient webClient;
+    @Autowired
```

Comment:
Looks like you have reformatted the code with a different formatter than the one that was agreed to be used. Please align with the team about which formatter you are using, to ensure consistent formatting.